docs/sources/structure/topic-types/visualization/index.md

```diff
@@ -0,0 +1,121 @@
+---
+title: Visualization topic
+menuTitle: Visualization
+description: Learn how to write a visualization topic.
+weight: 400
+aliases:
+  - /docs/writers-toolkit/structure/topic-types/visualization
+keywords:
+  - topic types
+  - template
+  - visualization
+---
+
+# Visualization topic
+
+A _visualization_ topic provides an overview of a visualization type, its use cases, and the display options available in the user interface (UI). Visualization topics always include conceptual and reference content and may include task content.
+
+The following types of content can be included in visualization topics:
+
+- A high-level overview of a visualization's features, with use cases
+- Descriptions of the UI options unique to that visualization (common options are documented separately)
+- Screenshots that help users understand UI interactions or hard-to-describe UI
+- As needed, guidance on how to use the visualization in special use cases
+- Best practice guidelines
+
+A visualization topic does not include:
+
+- Tutorial content
+
+## Visualization topic structure
+
+The following applies to visualizations that require a data source. For visualizations that don't require one, refer to [Widget topic structure](#widget-topic-structure).
+
+A visualization topic includes the following elements:
+
+- **Topic title:** A visualization topic title is the same as the name of the visualization as it's written in the UI (for example, "Time series"). Don't add the word "panel" or "visualization" to the title.
+- **Introduction:** Include an introduction that explains what the visualization type can help the user do.
+- **Main image:** Add a clear screenshot depicting the visualization displaying a typical use case.
+- **Body:** The body may include task/how-to information as needed. The body must include the following sections:
+
+  - **Supported data formats:** Provide guidance about which data formats are supported by the visualization type, with example use cases.
+  - **[Section] options:** For each drop-down section of options in the edit panel, add a section in the topic with the name of the UI section, followed by the word "options." For example, if a drop-down section of options is called "Axis" in the UI, include a section in the topic called "Axis options". This doesn't include the sections linked from the **Other options** section.
+  - **Other visualization options:** Include a section for links to the documentation for common UI options:
+
+    - Panel options
+    - Standard options
+    - Thresholds
+    - Legend
+    - Data links
+    - Field overrides
+    - Value mappings
+
+![Visualization structure](/media/docs/writers-toolkit/visualization-topic-example-ann-2.png)
+
+### Widget topic structure
```

Comment:
I wonder if the concept of a widget topic should be addressed in the H1, and then this could be an H2?

Reply:
Yeah my initial instinct when I did this section was to make this an H2, but it would require rewriting, but you've confirmed that was the right approach. I'll update and push the change separate from the other smaller changes you've suggested.

Reply:
👍